dappnode_package.json

```diff
@@ -2,6 +2,8 @@
   "name": "ethereum-classic.dnp.dappnode.eth",
   "version": "1.1.1",
   "upstreamVersion": "v1.12.6",
+  "upstreamRepo": "ethereum/go-ethereum",
```

Comment:
This needs to be reverted to https://github.com/etclabscore/core-geth

Reply:
Oh shit, right. @tropicar